### PR TITLE
add makeReplacements

### PR DESCRIPTION
**Pull request by michaelficarra**

### CODE REVIEW

File: run.js [hunk 1]
```diff
@@ -265,6 +265,25 @@ async function initGdocsClient(documentId) {
...
+  [/\b(a)ssessor\b/gi, (text, a) => `${a}ccessor`], // Ron
+  [/\bsho(?:e|ot?)\b/gi, 'SYG'], // Shu
+  [/\ba sink\b/gi, 'async'],
+  [/\bsink\b/gi, 'sync'],
+  [/\bDominic\b/g, 'Domenic'],
+  [/\bapi(s)\b/g, (text, a) => `API${s}`],
+]
+
+function makeReplacements(text) {
+  for (args of REPLACEMENTS) {
```
Comment (bakkot) [inline run.js]:
```suggestion
  for (let args of REPLACEMENTS) {
```

File: run.js [hunk 2]
```diff
@@ -265,6 +265,28 @@ async function initGdocsClient(documentId) {
...
+  [/javascript/gi, 'JavaScript'],
+  [/\bC[- ]sharp\b/gi, 'C#'], // Ron
+  [/\b(a)ssessor\b/gi, (text, a) => `${a}ccessor`], // Ron
+  [/\bsho(?:e|ot?)\b/gi, 'SYG'], // Shu
+  [/\ba sink\b/gi, 'async'],
+  [/\bsink\b/gi, 'sync'],
+  [/\bdominic\b/gi, 'Domenic'],
+  [/\bapi(s)\b/g, (text, s) => `API${s}`],
+  [/\bequal system(s)\b/g, (text, s) => `ecosystem${s}`],
+  [/\bdom\b/gi, 'DOM'],
```
Comment (robpalme) [inline run.js]: Dome -> DOM

**Review by robpalme — APPROVED** [review]

File: run.js (hunk 2 above)
Comment (michaelficarra) [reply run.js]: Added.

**Comment by bakkot** [comment]
I'll pull this in before tonight. A couple of others:

- jessie -> JS
- economic -> ergonomic

**Comment by michaelficarra** [comment]
@bakkot Added.

**Comment by michaelficarra** [comment]
This is now all in the `tc39` branch. Closing.

**Pull request closed without merging**